manifests/profile.pp

```diff
@@ -86,6 +92,8 @@
   $cron_enabled        = $duplicity::cron_enabled,
   $cron_hour           = undef,
   $cron_minute         = undef,
+  $duply_executable    = $duplicity::real_duply_executable,
```

Comment:
I've removed the `real_duply_executable` on master as it was causing some trouble (#10) and made things unnecessarily complex. Instead, the `PATH` environment variable is set.

I suggest to change the default of the parameter to `duply`.